### PR TITLE
add TC_00.001.07 | Verify that duplicate names are not accepted

### PR DESCRIPTION
**Pull request by ols0test**
[TC_00.001.07](https://github.com/orgs/RedRoverSchool/projects/4/views/1?pane=issue&itemId=87727843&issue=RedRoverSchool%7CJenkinsQA_JS_2024_fall%7C126)

### CODE REVIEW

File: cypress/e2e/createFreestyleProject.cy.js [hunk 1]
```diff
@@ -70,4 +70,27 @@ describe('US_00.001 | New item > Create Freestyle Project', () => {
...
+        cy.get('button[name="Submit"]').click();
+        cy.get('li[class="jenkins-breadcrumbs__list-item"]').eq(0).click();
+        cy.get('a[href$="/newJob"]').click();
+        cy.get('input#name').type(projectName);
+        cy.get('#items li[class$="FreeStyleProject"]').click();
+
+        cy.get('div[class$="validation-message"]').should('have.text', duplicateNameError);
+        cy.get('button[id="ok-button"]')
+          .should('be.disabled')
+          .should('be.visible');
```
Comment (Rom-T) [inline cypress/e2e/createFreestyleProject.cy.js:93]: instead of the second `should()` the `and()` method is usually used

**Review by adiuzhikov — APPROVED** [review]
Molodec!

**Review by ols0test — COMMENTED** [review]
Removed .only from the test and changed second .should to .and

File: cypress/e2e/createFreestyleProject.cy.js (hunk 1 above)
Comment (ols0test) [reply cypress/e2e/createFreestyleProject.cy.js]: Good point @Rom-T I totally forgot about that

**Pull request merged**